Dockerfile

```diff
@@ -50,7 +50,23 @@ WORKDIR /app
 
 RUN cargo install --path ./radix-clis
 
-FROM base-image
+# This dev-container image can be built with the following command:
+# docker build . --target scrypto-dev-container -t scrypto-dev-container
+FROM base-image AS scrypto-dev-container
+RUN apt install -y curl bash-completion git
+# Install improved prompt for better dev experience - https://starship.rs/
+RUN curl -sS https://starship.rs/install.sh | sh -s -- -y
+RUN echo 'eval "$(starship init bash)"\n . /etc/bash_completion' >> /root/.bashrc
+
+COPY --from=builder /app/target/release/scrypto /usr/local/bin/scrypto
+COPY --from=builder /app/target/release/resim /usr/local/bin/resim
+COPY --from=builder /app/target/release/rtmc /usr/local/bin/rtmc
+COPY --from=builder /app/target/release/rtmd /usr/local/bin/rtmd
+COPY --from=builder /app/target/release/scrypto-bindgen /usr/local/bin/scrypto-bindgen
+RUN rustup target add wasm32-unknown-unknown
+RUN rustup component add rustfmt
```

Comment:
Ohh nice! We should probably add clippy too?

Reply:
Done 👍🏽